### PR TITLE
multi-line comments formatting

### PR DESCRIPTION
**Pull request by 19ngaddam**

### CODE REVIEW

File: public/convert.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```
Comment (zlister) [inline public/convert.html]: I think this file may be leftover from the TrialScope implementation

File: src/env.ts [hunk 2]
```diff
@@ -11,7 +12,7 @@ const defaults: {[key: string]: string | null} = {
 };
 
 
-//********* Potentially trialscope specific code ********
+//* Potentially trialscope specific code *
```
Comment (zlister) [inline src/env.ts]: This code would be used if the API needs some sort of authorization. Not necessarily TrialScope specific

File: src/server.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        // console.log(JSON.stringify(fhirResult, null, 2));
+        res.status(200).send(JSON.stringify(fhirResult));
+      }).catch(error => {
+        console.error(error);
+        res.status(500).send({ error: 'Error from server', exception: Object.prototype.toString.call(error) as string });
+      });
+    } else {
+      res.status(400).send({ error: 'Invalid patientBundle' });
+    }
+  } /* no support for raw searchBody now
```
Comment (zlister) [inline src/server.ts]: This commented out code can be deleted. There either is a bundle or there isn't.

**Review by zlister — CHANGES_REQUESTED** [review]

File: public/convert.html (hunk 1 above)
Comment (dmpotter44) [reply public/convert.html]: This file is definitely left over from TrialScope, it was used to test the convert condition code to TrialScope code service.

File: spec/support/server.spec.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+describe("server tests", () => {
+  it("responds to /", () => {
+    return request(server)
+      .get('/')
+      .set('Accept', 'application/json')
+      .expect(200);
+  });
+
+  it("responds to / with hello from clinical trial ", () => {
```
Comment (dmpotter44) [inline spec/support/server.spec.ts]: We should probably remove the extra spaces from the end of the test descriptions.

**Review by zlister — DISMISSED** [review]

**Review by zlister — APPROVED** [review]

**Review by lelevine — APPROVED** [review]

**Review by dmpotter44 — APPROVED** [review]

**Pull request merged**